packages/lambda-cog/bundle.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#
+# Create a deployment bundle with sharp/libvips prebuilt included
+#
+../../scripts/bundle.js package.json
+cd dist
```

Comment:
We do not include any native modules so we shouldn't need this.

Reply:
If we remove the bundle, do we also need to update the code path to point to build instead of dist? `const CODE_PATH = '../lambda-cog/build;`

Reply:
it should be bundled into `dist` for us by the bundler.

What happens when you run bundle in the root?
```
npx lerna run bundle
```

`lambda-analytics` would be a better example as it hast a lot less stuff in it.

Reply:
I see, just fixed that, I point to the correct bundle script in the package.json.